### ISSUE TITLE
not currently able to mint new purls

### ISSUE THREAD

**Issue by philbarker** [dcmi/LRMI_Vocabs#42]
The URIs for the terms are in the PURL namespace, however problems with the OCLC purl service mean that it is currently not possible mint PURLs [1].

Options:
1. Release with PURLs which will not resolve, in the hope and expectation that the PURL service gets fixed.
2. Change to another namespace that we hope and expect will be persistent.
3. Delay release until the situation clarifies.

While we still have other unresolved issues, option 3 seems the way to go.

1. https://www.jiscmail.ac.uk/cgi-bin/webadmin?A2=ind1511&L=DC-ARCHITECTURE&F=&S=&P=3711

**Comment by stuartasutton** [comment]
Yup. I agree. DCMI will do what it can to assist in finding an appropriate solution.
